imports/ui/components/RefreshButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import Button from "@material-ui/core/Button";
+import Renew from "@material-ui/icons/Autorenew";
+import "./RefreshButtonStyle.css"
+
+class RefreshButton extends Component {
```

Comment:
This probably doesn't need to be a class, but that's fine.

Reply:
Okayyyy